Branch-SDK/src/main/java/io/branch/referral/PrefHelper.java

```diff
@@ -54,7 +54,9 @@
      */
     private static final int MAX_RETRIES = 3; // Default retry count is 3
 
-    static final int TIMEOUT = 5500; // Default timeout id 5.5 sec
+    static final int TIMEOUT = 5500; // Default timeout is 5.5 sec
+    static final int CONNECT_TIMEOUT = 10000; // Default timeout is 10 seconds
+    static final int TASK_TIMEOUT = TIMEOUT+CONNECT_TIMEOUT; // Default timeout is 15.5 seconds
```

Comment:
Since this is computed as a constant, I would do the same for the setters.

Reply:
Done